src/components/organisms/Form/index.tsx

```diff
@@ -7,9 +7,9 @@ import "./styles.css";
 
 export const Form = () => (
   <form action="#" method="post">
-    <Input type="email" placeholder="E-mail" />
-    <Input type="password" placeholder="Password" />
-    <Input type="password" placeholder="Repeat password" />
+    <Input type="submit" placeholder="E-mail" />
+    <Input type="submit" placeholder="Password" />
+    <Input type="submit" placeholder="Repeat password" />
```

Comment:
This is totally incorrect. You had correct types and now change them to incorrect ones.